src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-subcomponent-interactor/table-subcomponent-interactor.component.html

```diff
@@ -0,0 +1,12 @@
+<div [ngClass]="displayTopLineClass"></div>
+<cp-table-interactor-stoichiometry
+  *ngIf="interactorInComplex"
+  [stoichiometryText]="interactorStoichiometryText"
```

Comment:
Not completely necessary, but the same thing we did in this component, having the values in variables set in ngOnChangesn and not calling methods in ngIf; we could do in other components too.